samples/application-conf/application.properties

```diff
@@ -58,46 +58,50 @@ jobCard=
+#     - See: https://www.ibm.com/docs/en/dbb/1.1.0?topic=scripts-how-organize-build-script#file-properties
+#   - Overwrites for an individual file
+#     - Defined in an individual property file located in a configurable subfolder (e.g. properties/epsmlist.cbl.properties)
+#
+# A typical scenario for using zAppBuild's capability to set build properties for an individual source file via a corresponding
+# individual property file is to overwrite compile or link options. This approach might help ease the migration of properties
+# from the previous build system.
+#
+# Individual property files are resolved using the pattern <propertyFilePath directory>/<sourceFile>.<propertyFileExtension>. For example,
+#  for the source file epsmlist.cbl, the process searches for a file in the propertyFilePath directory
+#  with the name epsmlist.cbl.properties.
+#  If no corresponding property file is found, the build will use the default build values. (Or, if any file properties were defined
+#  using the DBB file property path syntax, then the build will use those.)
+#
+# Note: Overwrites for a specific build property should be managed either via the file property path syntax or
+#  in the individual property files, but not both. The following example shows how both approaches for defining
+#  file properties can be combined to specify a set of build properties for the same source file:
+#
+# ### Example: Using the file property path syntax and individual property files to define build properties for a source file named app/cobol/AB123456.cbl
+# - You can use the file property path syntax to define a file property for a group of files. The below defines the deployType for
+#  all source files in the folder cobol beginning with AB* to be BATCHLOAD:
 #
 #  cobol_deployType = BATCHLOAD :: **/cobol/AB*.cbl
 #
-# While you define a individual file property file for app/cobol/AB123456.cbl containing
+# - At the same time, you can define an individual file property file for app/cobol/AB123456.cbl with the following build property:
 #  
-#  cobol_compileParms = LIB,SOURCE   
+#  cobol_compileParms = LIB,SOURCE
+# 
+# - During the build, the file app/cobol/AB123456.cbl will have the deployType BATCHLOAD and the COBOL compile parameters LIB and SOURCE.
+# ### End example ###
 
-#
-# flag to enable the zAppBuild capability to load individual property files for a build file, default: false
-# It is recommended to use a file property path syntax to activate the loading of property files for a group of files,
-# while not all types in the repository. See above sample. 
+# ### Properties to enable and configure build property overwrites using individual property files
+
+# flag to enable the zAppBuild capability to load individual property files for individual source files, default: false
+# Note: To activate the loading of property files for a group of files, it is recommended to use the file property path
+# syntax instead. (See above example with **/cobol/AB*.cbl.)
```

Comment:
The original wording for this line was:
> It is recommended to use a file property path syntax to activate the loading of property files for a group of files, while not all types in the repository.

Can you please check if I have preserved the original meaning of this sentence? I am not sure about the last part ("while not all types in the repository").

Reply:
Perfect wording!